core/src/main/scala/wartremover/contrib/warts/SymbolicName.scala

```diff
@@ -0,0 +1,27 @@
+package org.wartremover
+package contrib.warts
+
+object SymbolicName extends WartTraverser {
+
+  private val validChars = "[a-zA-Z_]+".r
+
+  def apply(u: WartUniverse): u.Traverser = {
+    import u.universe._
+
+    def isSymbolic(name: String) = validChars.replaceAllIn(name, "").length > 2
```

Comment:
I think the badge is borrowed from JavaScript. True the badge is not perfect but that is what is used and what Scala maintainers support